src/beamlime/dashboard/dashboard.py

```diff
@@ -185,17 +192,21 @@ def start_periodic_updates(self, period: int = 500) -> None:
             second, this default should reduce UI lag somewhat. If there are no new
             messages, the step function should not do anything.
         """
-        if self._callback is None:
```

Comment:
Previously this failed to recreate a callback, so plots did not update any more after pressing "reload" in the browser. Unrelated fix to this PR.